guide/installation-and-configuration/upgrading.markdown

```diff
@@ -9,10 +9,15 @@ This guide documents our recommendation on how to upgrade an existing
 installation of CFEngine Community 3.7 and CFEngine Enterprise 3.7 to
 CFEngine {{site.cfengine.branch}}.
 
-We reccomend upgrading the Masterfiles Policy Framework first so that you can
+We recommend upgrading the Masterfiles Policy Framework first so that you can
 verify that the new policy will work with your old clients across the
 infrastructure. Once the latest policy has been deployed successfully we
-reccomend upgrading the Policy Server and finally the remote agents.
+recommend upgrading the Policy Server and finally the remote agents.
+
+Upgrading to {{site.cfengine.branch}} from versions older than 3.7 is more
+complicated as some functionalities introduced in {{site.cfengine.branch}}
+are not compatible with versions 3.6 and earlier. 
+For more information about upgrading from 3.6 see [Upgrade from 3.6](#upgrade-from-36).
```

Comment:
as some *functionality* introduced in {{site.cfengine.branch}}
*is* not compatible